twake/frontend/src/app/components/search-popup/parts/recent/files-result.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { ChannelType } from 'features/channels/types/channel';
+import { FileType } from 'features/files/types/file';
+
+const locale = navigator.languages[0];
+
+type PropsType = {
+  file: FileType;
+  onDownloadClick: any;
+  onPreviewClick: any;
+};
+
+const iconFileByMime = (mimetype: string) => {
+  const type =
+    (~mimetype.indexOf('spreadsheetml') && 'xls') ||
+    (~mimetype.indexOf('msword') && 'doc') ||
+    (~mimetype.indexOf('pdf') && 'pdf') ||
+    (~mimetype.indexOf('zip') && 'zip') ||
+    'unknown';
+
+  return `/public/icons/file-type-${type}.svg`;
```

Comment:
Please use FileUploadAPIClient.mimeToType() instead ;)

Reply:
Nice! Btw we don't have type for doc